src/group_theory/subgroup.lean

```diff
@@ -719,6 +729,14 @@ lemma map_supr {ι : Sort*} (f : G →* N) (s : ι → subgroup G) :
   (supr s).map f = ⨆ i, (s i).map f :=
 (gc_map_comap f).l_supr
 
+@[to_additive] lemma comap_sup
+  (H K : subgroup N) (f : G →* N) : comap f H ⊔ comap f K ≤ comap f (H ⊔ K) :=
+sup_le (comap_mono le_sup_left) (comap_mono le_sup_right)
```

Comment:
Untested, but
```suggestion
monotone.le_map_sup comap_mono H K
```

Reply:
Did this not work?

Reply:
Nope, lean says that it expects `monotone (comap f)` instead of  `K ≤ K' → comap f K ≤ comap f K'`.

Reply:
How about `monotone.le_map_sup (λ _ _, comap_mono) H K`?

Reply:
Ah yes! It works